src/app/components/Header/Header.tsx

```diff
@@ -6,8 +6,33 @@ import classNames from 'classnames';
 import { basePath } from '../../utils/constants';
 import { NavigationLink } from '../NavigationLink/NavigationLink';
 import styles from './Header.module.scss';
+import Hero from '../Hero/Hero';
+import { usePathname } from 'next/navigation';
 
 export default function Header() {
+  interface HeroContent {
+    heroClass?: string;
+    heroHeader: string;
+    heroSubheader: string;
+  }
+
+  type SpecialContent = {
+    [key: string]: HeroContent;
+  };
```

Comment:
Right now TypeScript shows `customContent` as always being defined, which may not be true. You correctly added the `customContent &&` before rendering the `Hero` but TS is telling me it's fine without that check, which isn't right.

We can modify this type slightly so that it can be `undefined`:
```suggestion
  type SpecialContent = Record<string, HeroContent | undefined>;
```

Reply:
done